po/zh_CN.po

```diff
@@ -703,3 +703,31 @@ msgstr "正在监听端口 %d"
 
 #~ msgid "Could not create framebuffer image"
 #~ msgstr "无法创建帧缓冲图像"
+
+#: unix/vncpasswd/vncpasswd.cxx:121
+msgid "Cannot check password quality: %s\n"
+msgstr "不能检查密码复杂度 %s\n"
+
+#: unix/vncpasswd/vncpasswd.cxx:134
+msgid "Password quality check failed:\n %s \n"
+msgstr "密码复杂度检查失败 \n %s \n"
+
+#: unix/vncpasswd/vncpasswd.cxx:154
+msgid "Password not changed\n"
+msgstr "密码没有修改\n"
+
+#: unix/vncpasswd/vncpasswd.cxx:157
+msgid "Password must be at least 6 characters - try again\n"
+msgstr "密码应至少6位,请重试\n"
+
+#: unix/vncpasswd/vncpasswd.cxx:162
+msgid "Password should not be greater than 8 characters\nBecause only 8 valid characters are used - try again\n"
+msgstr "密码长度不应该大于8位,因为仅只有8位被使用,请重试\n"
+
+#: unix/vncpasswd/vncpasswd.cxx:169
+msgid "Password quality check failed, please set it correctly.\n"
+msgstr "密码复杂度检查失败,请设置正确的密码\n"
+
+#: unix/vncpasswd/vncpasswd.cxx:181
+msgid "Passwords don't match - try again\n"
+msgstr "密码不匹配,请重试\n"
```

Comment:
I'm afraid our translations are handled by the translation project. As a result, we cannot modify these files directly or they'll get overwritten. Please send your translations to them once a beta is out of TigerVNC with your changes in it.

Reply:
Okay, I understand. Maybe I didn't consider it before.